.env.example

```
@@ -26,6 +26,12 @@ GITHUB_TOKEN=ghp_...  # Same as GH_TOKEN, used by adapter
 
 # GitHub Webhooks
 WEBHOOK_SECRET=your_random_secret_string
+WEBHOOK_REPO_MUTATING_BUDGET_LIMIT=25
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The WEBHOOK_REPO_MUTATING_BUDGET_LIMIT key should go before the WEBHOOK_SECRET key

(UnorderedKey)